Assets/MRTK/Core/Providers/InputSimulation/InputSimulationService.cs

```diff
@@ -130,11 +130,18 @@ public void ResetHandRight()
                 handDataProvider.ResetHand(Handedness.Right);
             }
         }
+        
+        private EyeGazeSimulationMode eyeGazeSimulationMode;
+        /// <inheritdoc />
+        public EyeGazeSimulationMode EyeGazeSimulationMode
+        {
+            get => eyeGazeSimulationMode;
+            set
+            {
+                eyeGazeSimulationMode = value;
+            }
+        }
 
-        /// <summary>
-        /// If true then camera forward direction is used to simulate eye tracking data.    
-        /// </summary>
-        public bool SimulateEyePosition { get; set; }
```

Comment:
Also nit: Similar to the previous probably want to preserve this with an Obsolete tag, and then map to the new one in whatever way you choose with the other comment